spec/ruby/core/enumerable/each_slice_spec.rb

```diff
@@ -56,4 +56,10 @@
     multi = EnumerableSpecs::YieldsMulti.new
     multi.each_slice(2).to_a.should == [[[1, 2], [3, 4, 5]], [[6, 7, 8, 9]]]
   end
+
+  it "has correct size with no proc given" do
```

Comment:
Spec descriptions should read as natural English. While quite a few existing descriptions don't follow this guideline we try to apply it to new changes. As such I'd suggest writing this something along the lines of `it "returns the correct size when no block is given"`.